### DISCUSSION TITLE
How to Combine a MP3 files from PROGMEM and A2DP using OutputMixer?

### DISCUSSION THREAD

**Discussion by timeylies** [pschatzmann/arduino-audio-tools#713] [Q&A] [answered]
I'm trying to combine multiple MP3 files and a2dp with a mixer so I can play status sounds while playing music through a2dp

**Comment by pschatzmann** [comment, marked as answer]
I would try it with an input mixer first. But you forgot to tell me if you implement an A2DP source or a sink...
Don't forget that you can only mix pcm audio, so you need to decode your mp3 files first!

**Comment by pschatzmann** [comment]
Still the same answer: since you can read data from the A2DPStream and the EncodedAudioStream you can use the InputMixer

**Reply by timeylies** [reply]
Great thank you
